assets/scss/_variables_project.scss

```diff
@@ -37,8 +37,17 @@ $table-row-alternate: $secondary-lightest-background;
 $table-row-default: $secondary-lighter-background;
 
 /*
- Opt to serve fonts locally by overriding web-font-path to be a non-google fonts URL.
- This is only possible with our modified docsy theme: https://github.com/matrix-org/docsy
+ Disable loading any fonts from Google Fonts.
  */
-$web-font-path: "../css/fonts/Inter.css";
-$google_font_name: "Inter";
+$td-enable-google-fonts: false;
+
+/*
+ Replace the default font with Inter - we load it from a local copy, which is downloaded from
+ Google Fonts manually via a script:
+ https://github.com/matrix-org/matrix-spec/tree/main/static/css/fonts
+
+ The list of fonts (with Inter prepended) is taken from docsy's source code:
+ https://github.com/matrix-org/docsy/blob/66a4e61d2d34edc7196b9df83a7d09cd4af14b47/assets/scss/_variables.scss#L68
+ */
+@import url("../css/fonts/Inter.css");
+$font-family-sans-serif: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
is this `$font-family-sans-serif` ours? or bootstrap's? Or docsy's? Or something else?

Reply:
Hopefully answered above.